### PR TITLE
[TW-843] flows auth tutorial

### PR DESCRIPTION
**Pull request by petewitcher**

### CODE REVIEW

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* [Create a secret key for the API](#create-a-secret-key-for-the-api)
+* [Create a variable for the secret key](#create-a-variable-for-the-secret-key)
+* [Create a collection and a request](#create-a-collection-and-a-request)
+* [Create a flow](#create-a-flow)
+
+## Create a secret key for the API
+
+This example flow uses a secret key to access the OpenAI API. To get your secret key:
+
+1. In your browser, go to https://beta.openai.com/account/api-keys and select **Sign up**.
```
Comment (jkonrath-postman) [inline src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:19]: Either make this a live link, or put it in a code font

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+1. Follow the prompts to create and sign into an OpenAI account.
+1. Return to https://beta.openai.com/account/api-keys and create a secret key.
```
Comment (jkonrath-postman) [inline src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:21]: Same as above

**Review by jkonrath-postman — APPROVED** [review]
Looks great! I'm approving even though I didn't get a chance to test it - make sure someone else does before you merge.

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md (hunk 1 above)
Comment (petewitcher) [reply src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:19]: Made it a live link.

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md (hunk 2 above)
Comment (petewitcher) [reply src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:21]: Made it a live link.

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Using authorization with Postman Flows"
+updated: 2022-01-10
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:3]:
```suggestion
updated: 2023-01-10
```

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+* [Create a variable for the secret key](#create-a-variable-for-the-secret-key)
+* [Create a collection and a request](#create-a-collection-and-a-request)
+* [Create a flow](#create-a-flow)
+
+## Create a secret key for the API
+
+This example flow uses a secret key to access the OpenAI API. To get your secret key:
+
+1. In your browser, go to [https://beta.openai.com/account/api-keys](https://beta.openai.com/account/api-keys) and select **Sign up**.
+1. Follow the prompts to create and sign into an OpenAI account.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:20]:
```suggestion
1. Follow the prompts to create and sign in to an OpenAI account.
```

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+1. Under **TYPE**, select **secret** from the dropdown list.
+
+    ![Create a variable](https://assets.postman.com/postman-docs/v10/flow-auth-create-variable-v10.jpg)
+
+1. Select <img alt="Save icon" src="https://assets.postman.com/postman-docs/icon-save.jpg#icon" width="16px"> **Save**.
+1. Close the **Globals** tab.
+
+## Create a collection and a request
+
+1. [Create a collection](/docs/getting-started/creating-the-first-collection/) named OpenAI Collection and add a GET request with this URL: `https://api.openai.com/v1/models`.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:40]:
```suggestion
1. [Create a collection](/docs/getting-started/creating-the-first-collection/) named `OpenAI Collection` and add a GET request with this URL: `https://api.openai.com/v1/models`.
```

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+1. Return to [https://beta.openai.com/account/api-keys](https://beta.openai.com/account/api-keys) and create a secret key.
+1. Copy and paste your secret key in a secure but accessible location. You'll use it in a later step.
+
+## Create a variable for the secret key
+
+By assigning your secret key value to a variable, you can keep your key secure and easily include it in multiple **Send Request** blocks and flows.
+
+1. In  your Postman workspace, select the environment quick look icon <img alt="Environment quick look icon" src="https://assets.postman.com/postman-docs/icon-environment-quick-look.jpg#icon" width="16px"> in the [workbench](/docs/getting-started/navigating-postman/#environment-selector-and-environment-quick-look).
+1. Next to **Globals**, select **Edit** (or **Add**).
+1. Add a variable named `auth` and paste your secret key in the **INITIAL VALUE** field.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:30]: When I entered the API key as is and then sent the request, Postman returned a `401` status code.

Looking at [OpenAI's documentation](https://beta.openai.com/docs/api-reference/requesting-organization), the API needs to be in the form `Bearer YOUR_API_KEY`. After I entered `Bearer` before the API key in my secret variable, I was able to send the request and return a `200` status code.

I'm not sure if others had the same experience.

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+## Create a flow
+
+1. [Create a new flow](/docs/postman-flows/flows-intro/building-your-first-flow/) and connect a **Send Request** block to the **Start** block.
+1. In the **Send Request** block, select **Add request > OpenAI Collection > New Request**. The `{{auth}}` variable appears at the bottom of the **Send Request** block.
+
+    ![Select data](https://assets.postman.com/postman-docs/v10/flow-auth-send-request-v10.jpg)
+
+1. Connect a **Log** block to the **Send Request** block's **Success** port.
+1. Select **Run** then select **Console**. The flow gets your secret key from the `{{auth}}` variable and includes it in the API request. The API accepts the secret key and sends the response with a `200 OK` code, visible in the console.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:64]: In the console, I only see `200` as the status code (in the top-right corner), not `200 OK`. Although, I'm testing in Production, so I might be seeing something different.

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good! I ran into an issue with the OpenAI secret key, but I'm not sure if this was specific to me.

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md (hunk 6 above)
Comment (petewitcher) [reply src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:30]: Good call, fixed.

File: src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md (hunk 7 above)
Comment (petewitcher) [reply src/pages/docs/postman-flows/flows-intro/using-auth-with-flows.md:64]: Fixed

**Pull request merged**